.pre-commit-config.yaml

```diff
@@ -40,15 +40,3 @@ repos:
         types: [go]
         language: golang
         pass_filenames: false
-  - repo: https://github.com/python-jsonschema/check-jsonschema
```

Comment:
Removing this as now that that we check the schema on create so the only time it will be invalid is if it's in a test